.travis.yml

```diff
@@ -5,8 +5,6 @@ python:
     - "3.8"
     - "3.7"
     - "3.6"
-    - "3.5"
-    - "2.7"
```

Comment:
These python versions should also be taken out of the `setup.py` file and the `README.rst` should be updated accordingly.

Reply:
Well spotted.

Reply:
stumbled over a python35 environment which didn't work with anesthetic anymore...

Reply:
Can you remember what it was that didn't work?

Reply:
The newest matplotlib (v3.1.2) which incorporates the vertical slider has dependency python>=3.6. 

Maybe worth incorporating in the requirements? i.e.
```
matplotlib>=3.1.2
```

Reply:
Added as part of #73